tests/integration/test_sqs.py

```diff
@@ -2830,3 +2837,67 @@ def test_get_without_query_json_format_returns_returns_xml(
 
     # TODO: write tests for making POST requests (not clear how signing would work without custom code)
     #  https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-making-api-requests.html#structure-post-request
+
+    @pytest.mark.aws_validated
+    def test_sqs_queue_as_lambda_dead_letter_queue(
+        self,
+        sqs_client,
+        lambda_client,
+        lambda_su_role,
+        create_lambda_function,
+        sqs_create_queue,
+        sqs_queue_arn,
+        snapshot,
+    ):
+        snapshot.add_transformer(
```

Comment:
it would be great if we could move this test into the lamdba sqs integration test (`test_lambda_sqs_integration.py`)

Reply:
Will do! On another note, we should be careful when renaming tests or moving them around, to think about cleaning up the snapshots. I guess it doesn't hurt, but having obsolete snapshots can maybe lead to small mistakes later on while looking at the files.